build-tools/packages/build-tools/src/common/tscUtils.ts

```diff
@@ -178,6 +213,7 @@ export function getTscUtil(tsLib: typeof ts) {
 			return configFile.config;
 		},
 		convertToOptionsWithAbsolutePath,
-		getCanonicalFileName: createGetCanonicalFileName(tsLib),
+		getCanonicalFileName: getCanonicalFileName,
+		getSourceFileVersion: getSourceFileVersion,
```

Comment:
opportunistic fix: we had already constructed the getCanonicalFileName above.  Reusing it here rather than constructing it again.